### ISSUE TITLE
Let Active-->Submitted state changes subsequently convert back to Active when only participant eligibility info has changed.

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#1001]
# TL;DR

As a researcher, I don't want my study to stop running when all I've done is to change the age criteria (often to narrow in on the end of a study sample.) And as a reviewer, I don't want to spend time re-approving changes that don't really need admin oversight!

# Narrative

When researchers edit a study and only change the eligibility criteria (including the family-facing language, criteria expression, and min/max age limits), that study should return to APPROVED without requiring admin review. 

# Acceptance Criteria

[ ] The following list of study fields should be exempt from requiring admin review: 

criteria,
max_age_months,
max_age_years,
min_age_months,
min_age_years,
max_age_days,
min_age_days,
criteria_expression

[ ] Text on the warning box that pops up when a user tries to save edits to a currently running study should mention that some minor changes will get automatically re-approved.  Right now it's a warning of some kind; mks is not certain what the exact current text of that popup is.

[ ] The study log table should somehow reflect the fact that this is an auto-approval rather than an admin approval. 

[ ] (@okaycj to update/refine these criteria)

# Implementation Notes

- We discussed implementing this as an ACTIVE-SUBMITTED followed by SUBMITTED-APPROVED set of state changes, rather than a complicated ACTIVE-APPROVED or ACTIVE-ACTIVE state change

- Proposal: Rename "APPROVED" to "ADMIN-APPROVED" and add "AUTO-APPROVED" as a new state?

**Comment by mekline** [comment]
Closing because #999 already exists
